frontend/src/components/header/Header.tsx

```diff
@@ -1,15 +1,22 @@
 import React, { useContext } from 'react';
 import { AuthContext } from '../../contexts/authContext';
-import { useNavigate } from 'react-router-dom';
-import './header.css';
+import { useNavigate, useLocation } from 'react-router-dom';
+import style from './header.module.css';
 import me from '../../assets/me.png';
 import useAdminStore from '../../store/admin.store';
 
 function Header() {
   const { logout, currentUser } = useContext(AuthContext);
   const resetUser = useAdminStore((state) => state.resetUser);
   const navigate = useNavigate();
+  const displayName = currentUser.displayName.split('   ');
 
+  const formattedFirstName =
+    displayName[0]?.charAt(0).toUpperCase() + displayName[0]?.slice(1).toLowerCase();
+  const formattedLastName =
+    displayName[1]?.charAt(0).toUpperCase() + displayName[1]?.slice(1).toLowerCase();
+
```

Comment:
This is optional but if you agree that the code is more readable and cleaner lets do it 💪 it will take a second and I will love to explain if something is not clear

Reply:
of course its much cleaner to do it your way , on it :grin: